calcom/cal.com
# fix: admin org list without members

## What does this PR do?

Fixing the list or orgs in admins settings, accounting for orgs without members

## Type of change


## How should this be tested?

Create a bunch of orgs and as an admin of the Cal.com instance see them in the list in settings.
